packages/calypso-e2e/src/lib/pages/themes-customizer-page.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Internal dependencies
+ */
+import { BaseContainer } from '../base-container';
+
+/**
+ * Type dependencies
+ */
+import { Frame } from 'playwright';
+
+const selectors = {
+	frame: 'iframe[title="Customizer"]',
+};
+
+/**
+ * Component representing the Appearance > Customize page.
+ *
+ * @augments {BaseContainer}
+ */
+export class ThemesCustomizerPage extends BaseContainer {
+	frame!: Frame;
+
+	/**
+	 * Post-initialization steps.
+	 *
+	 * @returns {Promise<void>} No return value.
+	 */
+	async _postInit(): Promise< void > {
+		const elementHandle = await this.page.waitForSelector( selectors.frame );
+		this.frame = ( await elementHandle.contentFrame() ) as Frame;
+	}
```

Comment:
Quick clarification: are we using `_postInit` here instead of passing that iframe selector into the constructor in order to stash off a pointer to the iframe?

I just want to make sure I am understanding the difference between the constructor and the selector passed there to be used during `Expect > _init()`, and `_postInit`

Reply:
Good observation! 

I am using `_postInit` to indeed save the page element referenced by the selector. This doesn't have to be done in `_postInit`, but I had wrongly assumed that all instances of the Customizer page would be wrapped in an iFrame. 

So this `_postInit` may not stay in the next revision, depending on how I decide to implement the page object for Customizer that can work with both iFrame wrapped and regular page instances.